services/api-server/src/simcore_service_api_server/api/routes/solvers_jobs_getters.py

```diff
@@ -334,7 +334,7 @@ async def get_job_output_logfile(
             f"{solver_key}/releases/{version}/jobs/{job_id}/outputs/logfile",
             presigned_download_link,
         )
-        return RedirectResponse(presigned_download_link)
+        return RedirectResponse(f"{presigned_download_link}")
```

Comment:
interesting that fastapi that is using pydantic v2 would not be compatible with new style URL? are you sure?

Reply:
```
src/simcore_service_api_server/api/routes/solvers_jobs_getters.py: note: In function "get_job_output_logfile":
src/simcore_service_api_server/api/routes/solvers_jobs_getters.py:337:33: error: Argument 1 to "RedirectResponse" has incompatible type "Url"; expected "str | URL"  [arg-type]
Found 1 error in 1 file (checked 90 source files)
```

```python
class RedirectResponse(
    url: str | URL,    # starlette.datastructures.URL
    status_code: int = 307,
    headers: Mapping[str, str] | None = None,
    background: BackgroundTask | None = None
)
```